### PR TITLE
Try/jetpack onboarding custom data type

### PR DESCRIPTION
**Pull request by JavonDavis**
Included custom data type as unique locators to reduce the use of sleeps in automated tests. Related to https://github.com/Automattic/wp-e2e-tests/issues/1196

### CODE REVIEW

**Comment by bsessions85** [comment]
Failed check ci/wp-e2e-tests-full-jetpack can be ignored.

File: client/components/wizard/progress-indicator.jsx [hunk 1]
```diff
@@ -9,7 +9,10 @@ import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 
 const ProgressIndicator = ( { stepNumber, totalSteps, translate } ) => (
-	<div className="wizard__progress-indicator">
+	<div
+		className="wizard__progress-indicator"
+		data-e2e-type={ 'step-indicator-' + ( stepNumber + 1 ) }
```
Comment (tyxla) [inline client/components/wizard/progress-indicator.jsx]: See https://github.com/Automattic/wp-e2e-tests/pull/1396/files#r208280875

File: client/jetpack-onboarding/steps/business-address.jsx [hunk 2]
```diff
@@ -208,7 +208,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 		const { getForwardUrl, hasBusinessAddress, siteId, translate } = this.props;
 
 		return (
-			<div className="steps__main">
+			<div className="steps__main" data-e2e-type="business-address">
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/business-address.jsx]: cc @Automattic/flowpatrol - is this a common pattern, or would it be easier if we introduce a classname like `steps__business-address` to each step instead?

File: client/jetpack-onboarding/steps/stats.jsx [hunk 3]
```diff
@@ -107,7 +107,7 @@ class JetpackOnboardingStatsStep extends React.Component {
 		const { activatedStats, getForwardUrl, siteId } = this.props;
 
 		return (
-			<div className="steps__main">
+			<div className="steps__main" data-e2e-type="activate-stats">
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/stats.jsx]: Is there any reason for this `data-e2e-type` to be different from the step name (`stats`)?

Comment (JavonDavis) [reply client/jetpack-onboarding/steps/stats.jsx]: No reason, I mapped this to the e2e test `ActivateStatsPage` class. To stay consist with calypso I'll send in a commit updating this and the e2e tests to use `stats`.

Comment (tyxla) [reply client/jetpack-onboarding/steps/stats.jsx]: Thank you 👍

**Pull request merged**